users/views.py

```diff
@@ -1,3 +1,35 @@
-from django.shortcuts import render
+import json, re, bcrypt
 
-# Create your views here.
+from django.http     import JsonResponse
+from django.views    import View
+
+from users.models import User
+
+class SignupView(View):
+    def post(self, request):
+        data             = json.loads(request.body)
+        email_validation = re.compile("^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$")
+        password_validation = re.compile("^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$")
```

Comment:
정규식을 사용할 때는 아래와 같은 형식으로 변수를 지정해주세요~

```suggestion
        REGEX_EMAIL = re.compile("^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$")
        REGEX_PASSWORD = re.compile("^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$")
```

Reply:
나머지는 완벽하네요~ 👍

Reply:
수정완료하였습니다!